taxi/models.py

```diff
@@ -0,0 +1,20 @@
+from django.conf import settings
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=63, unique=True)
+    country = models.CharField(max_length=63)
+
+
+class Car(models.Model):

```

Comment:
The `Meta` class is missing in the `Car` model. According to the checklist, you should always implement `Meta` class after you defined all attributes. The `Meta` class should contain `verbose_name` and `verbose_name_plural` in lowercase.